pegasus/sites.v3/code.org/views/homepage_gallery.haml

```diff
@@ -2,7 +2,7 @@
 - bgc_body = "255, 164, 0"        # "0,148,202" * 0.05
 
 #gallerycolumn.same-height-column
-  %a.nounderscorehover{href: "http://studio.code.org/gallery"}
+  %a.nounderscorehover{href: "https://studio.code.org/projects/public"}
```

Comment:
This should use `CDO.studio_url` so that it points to the correct gallery for the current environment.